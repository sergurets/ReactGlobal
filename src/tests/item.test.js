import React from 'react';
import { shallow } from 'enzyme';
import { shallowToJson } from 'enzyme-to-json';
import Item from '../components/item';

const data = [
    {
      budget:null,
      genres:[],
      id:null,
      overview:'',
      poster_path:null,
      release_date:"",
      revenue:null,
      runtime:null,
      tagline: null,
      title:null,
      vote_average:null,
      vote_count:null
    }      
  ]
describe('Item', () => {
  it('should render correctly', () => {
    const output = shallow(
      <Item data={data}/>
    );
    expect(shallowToJson(output)).toMatchSnapshot();
  });
});