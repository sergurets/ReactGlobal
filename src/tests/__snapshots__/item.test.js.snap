// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Item should render correctly 1`] = `
<div
  className="search_results"
>
  <div
    className="search_Item"
    key="null"
    onClick={[Function]}
  >
    <img
      src={null}
    />
    <ItemInfo
      genres={Array []}
      release_date=""
      title={null}
    />
  </div>
</div>
`;
